### DISCUSSION TITLE
Confirmación de Envió correcto de formulario

### DISCUSSION THREAD

**Discussion by emonreal9604** [LaunchX-InnovaccionVirtual/CursoIntroPython#252] [Q&A]
Hola buenas tardes,  ¿A alguien ya le llego la confirmación de que su envió fue correcto del formulario con las katas?.

**Comment by jstoledano** [comment]
A mi. Y ya estoy en el grupo de FrontEnd en teams.

**Reply by emonreal9604** [reply]
A que correo te llego? al de Launchx o al que te registraste? 
¿mas o menos cuanto te tardo en llegar?

**Reply by Ingrzs** [reply]
A mi me llego en cuestión de minutos quizas 2, y te llega al de innovación, abre outlook y mete usuario y contraseña que te dio sherpa

**Reply by Yvillacis** [reply]
Yo te sugiero que habras tu teams y revisa en la seccion de general no esta el link para el otro equipo. Yo tenia abierto el teams cuando envie el formulario y teams me notifico inmediatamente que habia un nuevo grupo para mi. Suerte.

**Reply by emonreal9604** [reply]
@Yvillacis Gracias compañera la volví enviar y ya me llego, supongo que como la primera vez la envié mientras tenia la VPN del trabajo tal vez por eso no me acepto el envió.

**Comment by AlejandroMontoyaT** [comment]
> Hola buenas tardes,  ¿A alguien ya le llego la confirmación de que su envió fue correcto del formulario con las katas?.

Te llega al correo de innovaccion

**Reply by emonreal9604** [reply]
Gracias compañero ya me llego

**Comment by doguedogue** [comment]
Una vez enviadas tus respuestas, recibirás un email a tu correo de Innovacción con la confirmación del estado de tu solicitud, en este momento manejamos 3 estados:
		
❌ Envío Incompleto - Registro Inválido: Sucede cuándo llenaste el formulario con una cuenta @innovaccion.mx pero que no es launchx es decir, no es una cuenta del programa oficial.	
❌ Envío Incompleto - Repositorio no válido: Verifica la escritura de tu enlace a GitHub, de lo contrario no podrás ingresar.
✔️ Envío Correcto - Bienvenid@ a la misión: A partir de ahora formas parte de la siguiente etapa del programa. No es necesario hacer algo más, serás añadid@ al nuevo equipo de Teams de forma automática. Te llegará el email de confirmación y la notificación en Teams.
	
Para revisar tu correo de innovacción debes:
Ingresar al [siguiente enlace](https://outlook.live.com/owa/)
Escribir tu correo de launchX
Escribir tu contraseña
Listo.

**Reply by emonreal9604** [reply]
Gracias compañero la volví enviar y ya me llego, supongo que como la primera vez la envié mientras tenia la VPN del trabajo tal vez por eso no me acepto el envió.
